scipy/ndimage/tests/test_filters.py

```diff
@@ -732,16 +752,16 @@ def test_gauss_axes_kwargs(self, axes):
         output = ndimage.gaussian_filter(array, axes=axes, **kwargs)
 
         # result should be equivalent to sigma=0.0 on unfiltered axes
-        sigma_3d = [0,] * array.ndim
+        sigma_3d = [0] * array.ndim
         sigma_3d[axes[0]] = sigma[0]
         sigma_3d[axes[1]] = sigma[1]
-        radius_3d = [0,] * array.ndim
+        radius_3d = [0] * array.ndim
         radius_3d[axes[0]] = radius[0]
         radius_3d[axes[1]] = radius[1]
-        mode_3d = [0,] * array.ndim
+        mode_3d = ['reflect'] * array.ndim
```

Comment:
Good catch.

Reply:
Yeah. It happened to work because it doesn't matter which mode is used since no padding actually occurs along non-filtered axes!